irf/utils.py

```diff
@@ -0,0 +1,12 @@
+def camera_distance_to_angle(theta):
```

Comment:
We cannot, because the more precise one needs knowledge of the distance to the camera center, we have just one arbitrary length